recipes/sdl/all/conanfile.py

```diff
@@ -176,19 +176,21 @@ def build(self):
         self._build_cmake()
 
     def _configure_cmake(self):
-        if not self._cmake:
-            self._cmake = CMake(self)
-            cmake_required_includes = []  # List of directories used by CheckIncludeFile (https://cmake.org/cmake/help/latest/module/CheckIncludeFile.html)
-            cmake_extra_ldflags = []
-            # FIXME: self.install_folder not defined? Neccessary?
-            self._cmake.definitions["CONAN_INSTALL_FOLDER"] = self.install_folder
-            if self.settings.os != "Windows":
-                if not self.options.shared:
-                    self._cmake.definitions["SDL_STATIC_PIC"] = self.options.fPIC
-            if self.settings.compiler == "Visual Studio" and not self.options.shared:
-                self._cmake.definitions["HAVE_LIBC"] = True
-            self._cmake.definitions["SDL_SHARED"] = self.options.shared
-            self._cmake.definitions["SDL_STATIC"] = not self.options.shared
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        cmake_required_includes = []  # List of directories used by CheckIncludeFile (https://cmake.org/cmake/help/latest/module/CheckIncludeFile.html)
+        cmake_extra_ldflags = []
+        # FIXME: self.install_folder not defined? Neccessary?
+        self._cmake.definitions["CONAN_INSTALL_FOLDER"] = self.install_folder
+        if self.settings.os != "Windows" and not self.options.shared:
+            self._cmake.definitions["SDL_STATIC_PIC"] = self.options.fPIC
+        if self.settings.compiler == "Visual Studio" and not self.options.shared:
```

Comment:
```suggestion
        if self.settings.compiler in ["Visual Studio", "msvc"] and not self.options.shared:
```

wdyt? Should we start considering `msvc` or maybe it is better to create a _tool_ to check for VS compiler?

Reply:
yes, there are 2 occurrences of Visual Studio, so it shouldn't be a big change.